source/adapters/level_zero/common.hpp

```diff
@@ -218,6 +218,13 @@ enum {
   UrL0SerializeBlock =
       2, // blocking UR calls, where supported (usually in enqueue commands)
 };
+typedef enum {
+  COUNTER_BASED = 1 << 0,
+  USING_IMM_CMDLIST = 1 << 1,
+  HOST_VISIBLE = 1 << 2,
+  ENABLE_PROFILER = 1 << 3,
+  MULTIDEVICE = 1 << 4
+} EventFlags;
```

Comment:
Storing invalid sequences in an enum is UB. You should have uint16_t or something to store the actual bit values, like in the UR API:
https://github.com/oneapi-src/unified-runtime/blob/main/include/ur_api.h#L552

Reply:
Done